torch/nn/modules/activation.py

```diff
@@ -1062,6 +1061,11 @@ def forward(self, query: Tensor, key: Tensor, value: Tensor, key_padding_mask: O
             `batch_first` argument is ignored for unbatched inputs.
         """
         is_batched = query.dim() == 3
+        if key_padding_mask is not None:
+            _skpm_dtype = key_padding_mask.dtype
```

Comment:
Variable name supposed to start with first letter of long variable name, i.e. for `key_padding_mask` is should be `_kpm_dtype`
```suggestion
            _kpm_dtype = key_padding_mask.dtype
```

Reply:
lol right